package.json

```diff
@@ -66,7 +66,9 @@
       "git add"
     ]
   },
-  "dependencies": {},
+  "dependencies": {
+    "@material-ui/styles": "^4.3.3"
```

Comment:
Looks like this got added at the workspace root, it should just be a dependency of the `component-library`. From the workspace root, can you `yarn remove @material-ui/styles`